lightning/src/ln/channel.rs

```diff
@@ -7162,8 +7153,6 @@ impl<SP: Deref> Writeable for Channel<SP> where SP::Target: SignerProvider {
 		// consider the stale state on reload.
 		0u8.write(writer)?;
 
-		self.context.funding_tx_confirmed_in.write(writer)?;
-		self.context.funding_tx_confirmation_height.write(writer)?;
```

Comment:
We'll need to maintain backwards compatibility. So even if internally we use a struct, we'll have to write out the fields individually.

That said, we could eventually bump the `SERIALIZATION_VERSION` to get around this, IIUC. We don't want to do that regularly, though. I seem to recall another recent PR where we talked about doing this, but I can't recall where.